### PR TITLE
Branch fix tooltip help issue

### PR DESCRIPTION
**Pull request by satabol**
Hello.

I'm beginner user of sverchok. I really like it. I read many nodes documentation and found some issues (I think about all nodes):

1. Several nodes link to other documentation. Ex. "Solid/Volume (Solid)" link to "Analyzers/Volume".
2. Some nodes and their docs has different headers.
3. Some errata.

## Solution description

1. I change some nodes names to capital letters like docs or docs like nodes names. (Except Nodes in Viz)
2. I change code to select correct documentation collection.

Is it to be helpfully?

If you decide to merge my changes - please check it. )))

### CODE REVIEW

File: docs/nodes/spatial/random_points_on_mesh.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Random points on mesh
+Populate Mesh (Random points on mesh)
 =====================
```
Comment (zeffii) [inline docs/nodes/spatial/random_points_on_mesh.rst:2]: extend the `=========` here..

File: docs/nodes/transforms/apply.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Matrix Apply
+Matrix Apply (verts)
 ============
```
Comment (zeffii) [inline docs/nodes/transforms/apply.rst:2]: and here..
```suggestion
===================
```

File: nodes/modifier_change/subdivide_mk2.py [hunk 3]
```diff
@@ -49,7 +49,7 @@ class SvSubdivideNodeMK2(
     '''
 
     bl_idname = 'SvSubdivideNodeMK2'
-    bl_label = 'Subdivide'
+    bl_label = 'Subdivide Node'
```
Comment (zeffii) [inline nodes/modifier_change/subdivide_mk2.py:52]: not sure we need `Node` here... think about it.

File: nodes/object_nodes/select_mesh_verts.py [hunk 4]
```diff
@@ -27,7 +27,7 @@
 class SvSelectMeshVerts(bpy.types.Node, SverchCustomTreeNode):
     ''' Select vertices of mesh objects '''
     bl_idname = 'SvSelectMeshVerts'
-    bl_label = 'Select Object Vertices'
+    bl_label = 'Select Vertices of Mesh'
```
Comment (zeffii) [inline nodes/object_nodes/select_mesh_verts.py:30]: This node really does place mesh elements of `scene.Objects` in a selected state, not sure if this is an improvement

File: old_nodes/mesh_select.py [hunk 5]
```diff
@@ -32,7 +32,7 @@
 class SvMeshSelectNode(bpy.types.Node, SverchCustomTreeNode):
     '''Select vertices, edges, faces by geometric criteria'''
     bl_idname = 'SvMeshSelectNode'
-    bl_label = 'Select mesh elements by location'
+    bl_label = 'Select Mesh Elements by location'
```
Comment (zeffii) [inline old_nodes/mesh_select.py:35]: why no cap for `Location` ? :)

**Comment by zeffii** [comment]
Thank you @satabol for your contribution, any help with the docs is always welcome - and this looks like considerable effort. I've commented by points of caution.

On top of that; I think you can fix the capitalization of our json templates in a separate PR. (or skip them altogether the effort may not be worth it.. but  up to you)

**Comment by zeffii** [comment]
the automated tests on the documenation says these undermarks are too short
```
List Mask (Out)
=============
/github/workspace/docs/nodes/spatial/random_points_on_mesh.rst:2: WARNING: Title underline too short.

Populate Mesh (Random points on mesh)
=====================
/github/workspace/docs/nodes/transforms/apply.rst:2: WARNING: Title underline too short.

Matrix Apply (verts)
============
```

File: docs/nodes/list_masks/mask.rst [hunk 6]
```diff
@@ -1,2 +1,2 @@
-List Mask Out
+List Mask (Out)
 =============
```
Comment (zeffii) [inline docs/nodes/list_masks/mask.rst:2]:
```suggestion
===============
```

File: old_nodes/mesh_select.py (hunk 5 above)
Comment (satabol) [reply old_nodes/mesh_select.py]: I skip it )))

File: docs/nodes/transforms/apply.rst [hunk 7]
```diff
@@ -1,5 +1,5 @@
-Matrix Apply
-============
+Matrix Apply (verts)
+===================
```
Comment (zeffii) [inline docs/nodes/transforms/apply.rst:2]:
```suggestion
====================
```

**Comment by satabol** [comment]
> On top of that; I think you can fix the capitalization of our json templates in a separate PR. (or skip them altogether the effort may not be worth it.. but up to you)
I agree with you. I queue this.

File: nodes/modifier_change/subdivide_mk2.py (hunk 3 above)
Comment (zeffii) [reply nodes/modifier_change/subdivide_mk2.py]:
```suggestion
Subdivide
```

**Review by zeffii — CHANGES_REQUESTED** [review]
we tend not to include `Node` in the bl_label of nodes.. it looks awkward.

**Comment by satabol** [comment]
> we tend not to include `Node` in the bl_label of nodes.. it looks awkward. in `Subdivide Node`

There is no 'Node' in bl_label:



Can you check last commit?

**Review by zeffii — APPROVED** [review]
this is better than before the PR,  time to commit :)  thank you @satabol

**Pull request merged**